### PR TITLE
Wiki Reorg: Funding Section

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

File: docs/general/funding.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+---
+
+import DocCardList from '@theme/DocCardList'; import MessageBox from "../../components/MessageBox";
+import "../../components/MessageBox.css";
+
+<MessageBox message="For fundraising in a decentralized, transparent, and regulatory compliant manner within the ecosystem, check out the [Polimec parachain](https://www.polimec.org/)." />
+
+:::tip
+
+Check out the [programs](programs-index) for all kinds of ecosystem support opportunities.
```
Comment (CrackTheCode016) [inline docs/general/funding.md]:
```suggestion
Check out the [programs](programs-index) for numerous ecosystem support opportunities.
```

File: docs/general/funding.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+:::
+
+<DocCardList />
+
+If you are certain you want to apply for a W3F grant, head straight to our
```
Comment (CrackTheCode016) [inline docs/general/funding.md]:
```suggestion
If you are certain you want to apply for a W3F grant, head straight to the
```

File: docs/general/funding.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    click H "https://wiki.polkadot.network/docs/en/learn-polkadot-opengov-treasury" "https://wiki.polkadot.network/docs/en/learn-polkadot-opengov-treasury" _blank
+    click J "https://wiki.polkadot.network/docs/decentralized-futures" "https://wiki.polkadot.network/docs/decentralized-futures" _blank
+```
+
+For a longer list and a description of the programs listed below, check out
+[our page on alternative funding opportunities](./grants.md#alternative-funding-sources).
+
+## Funding FAQ
+
+_Will the grant be awarded to those delivering a complete product that everyone in the ecosystem can
```
Comment (CrackTheCode016) [inline docs/general/funding.md]: Maybe these would be better as a numbered list, or similar?

**Review by CrackTheCode016 — COMMENTED** [review]
It overall looks fine to me, I do not have as many insights into the grants program and the like, so ultimately will wait for @semuelle to give the go-ahead on the details to do with funding.

**Comment by filippoweb3** [comment]
@semuelle can we merge this one?

**Comment by semuelle** [comment]
@PieWol Can you look into this?

**Comment by PieWol** [comment]
Hey @filippoweb3 ,
sorry for the delay. I'd like to come up with a new funding flowchart as we no longer accept any DF application. Hence it doesn't make sense to show this as a current funding source. I'll get back to you shortly.

**Comment by filippoweb3** [comment]
@PieWol any update?

File: docs/general/funding.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+## Funding Opportunities Overview
+
+```mermaid
+flowchart LR
+    A(Project Focus)
+    A -->|Development| B[Stage of Development]
+    A -->|Research| C[Grants Program]
+    A -->|Other| D[Business model exists]
+    B -->|Existing POC| E[Treasury or Decentralized Futures]
```
Comment (PieWol) [inline docs/general/funding.md]:
```suggestion
    B -->|Existing POC| E[Treasury]
```

File: docs/general/funding.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    style F stroke:#e83e8c,stroke-width:2px,stroke-dasharray: 5 5
+    style H stroke:#e83e8c,stroke-width:2px,stroke-dasharray: 5 5
+    style J stroke:#e83e8c,stroke-width:2px,stroke-dasharray: 5 5
+
+    click C "https://grants.web3.foundation/docs/Process/how-to-apply" "Apply now"
+    click E "https://wiki.polkadot.network/docs/decentralized-futures" "Apply Now"
+    click F "https://grants.web3.foundation/docs/Process/how-to-apply" "Apply now"
+    click H "https://wiki.polkadot.network/docs/en/learn-polkadot-opengov-treasury" "https://wiki.polkadot.network/docs/en/learn-polkadot-opengov-treasury" _blank
+    click J "https://wiki.polkadot.network/docs/decentralized-futures" "https://wiki.polkadot.network/docs/decentralized-futures" _blank
+```
```
Comment (PieWol) [inline docs/general/funding.md]:
```suggestion
    style C stroke:#e83e8c,stroke-width:2px,stroke-dasharray: 5 5
    style E stroke:#e83e8c,stroke-width:2px,stroke-dasharray: 5 5
    style F stroke:#e83e8c,stroke-width:2px,stroke-dasharray: 5 5
    style H stroke:#e83e8c,stroke-width:2px,stroke-dasharray: 5 5
    style J stroke:#e83e8c,stroke-width:2px,stroke-dasharray: 5 5

    click C "https://grants.web3.foundation/docs/Process/how-to-apply" "Apply now"
    click F "https://grants.web3.foundation/docs/Process/how-to-apply" "Apply now"
    click H "https://wiki.polkadot.network/docs/en/learn-polkadot-opengov-treasury" "https://wiki.polkadot.network/docs/en/learn-polkadot-opengov-treasury" _blank
```
```

File: docs/general/funding.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+```mermaid
+flowchart LR
+    A(Project Focus)
+    A -->|Development| B[Stage of Development]
+    A -->|Research| C[Grants Program]
+    A -->|Other| D[Business model exists]
+    B -->|Existing POC| E[Treasury or Decentralized Futures]
+    B -->|No POC| F[Grants Program]
+    D -->|No|H[Treasury]
+    D -->|Yes|J[Decentralized Futures]
```
Comment (PieWol) [inline docs/general/funding.md]:
```suggestion
    D -->|Yes|J[Venture Capital]
```

**Review by PieWol — CHANGES_REQUESTED** [review]
just some minor changes with decentralized futures being removed from the chart.

**Pull request merged**